cpplint/cpplint.py

```diff
@@ -861,6 +865,7 @@ def __init__(self):
     self._filters_backup = self.filters[:]
     self.counting = 'total'  # In what way are we counting errors?
     self.errors_by_category = {}  # string to int dict storing error counts
+    self.quiet = False # Suppress non-error messagess?
```

Comment:
The usual style (also used on the two lines above) is to put two spaces before the #.

Reply:
Solid catch. I fixed it. If only we had a pylint for our cpplint :).